src/core_plugins/kibana/public/dashboard/index.js

```diff
@@ -208,6 +208,17 @@ app.directive('dashboardApp', function (Notifier, courier, AppState, timefilter,
         chrome.addApplicationClass(theme);
       }
 
+      $scope.expandedPanel = null;
+      $scope.toggleExpandPanel = (panelIndex) => {
+        if ($scope.expandedPanel && $scope.expandedPanel.panelIndex === panelIndex) {
+          $scope.expandedPanel = null;
+        } else {
+          $scope.expandedPanel = $scope.state.panels.find(function (panel) {
```

Comment:
This could be simplified by using a fat arrow function:

```javascript
$scope.expandedPanel = $scope.state.panels.find(panel => panel.panelIndex === panelIndex);
```